### PR TITLE
create bblfsh-cli command

### PR DESCRIPTION
**Pull request by campoy**
This should be useful for anyone trying to give bblfsh a try from the command line.

Will it appear as a binary directly on the release, @smola?

### CODE REVIEW

**Comment by dennwc** [comment]
Maybe it's worth moving to SDK? @juanjux What do you think?

**Comment by juanjux** [comment]
The SDK is for development and the clients for using so I think this is the right place, the Python client already have this functionality.

File: cmd/bblfsh-cli/main.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+	"fmt"
+	"os"
+
+	flags "github.com/jessevdk/go-flags"
+	bblfsh "gopkg.in/bblfsh/client-go.v2"
+)
+
+func main() {
+	var opts struct {
+		Host     string `short:"h" long:"host" description:"Babelfish endpoint address" default:"35.204.62.43:9432"`
```
Comment (dennwc) [inline cmd/bblfsh-cli/main.go]: Please change the default

**Comment by bzz** [comment]
Would it make sense to include `-q XPATH` for UAST filtering as well?

Most probably everyone already knows this but just in case - this exists and is possible to do for a user with 2 other clients already:
 - [Scala](https://github.com/bblfsh/client-scala/blob/master/src/main/scala/org/bblfsh/client/cli/ScalaClientCLI.scala)    
 - [Python](https://github.com/bblfsh/client-python/blob/b64ff3a8a7928e163acfb61b76d84e92cea5de1d/bblfsh/__main__.py#L10)

**Comment by dennwc** [comment]
@bzz Yes, I think it would be nice. Could be a separate PR, though.

@campoy Will you implement this feature as well?

File: cmd/bblfsh-cli/main.go (hunk 1 above)
Comment (campoy) [reply cmd/bblfsh-cli/main.go]: To what, though? I'd like to point to a public bblfsh service which I requested on https://github.com/src-d/feature-idea/issues/45 but it's still not available.

Maybe I should simply drop the default and force the usage of some other IP?

**Comment by campoy** [comment]
Happy to implement it, it shouldn't be too hard given the right libraries.

**Comment by campoy** [comment]
Actually it was pretty straight forward, I added it to this PR.
Literally 12 lines of code

File: cmd/bblfsh-cli/main.go (hunk 1 above)
Comment (dennwc) [reply cmd/bblfsh-cli/main.go]: Sounds reasonable to drop it. Or maybe set to `localhost`?

Comment (juanjux) [reply cmd/bblfsh-cli/main.go]: `0.0.0.0:9432` is what's used everywhere (docs, tests, etc) as default.

Comment (campoy) [reply cmd/bblfsh-cli/main.go]: ok, I'll change it to localhost:9432

Comment (campoy) [reply cmd/bblfsh-cli/main.go]: done

**Pull request merged**